connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java

```diff
@@ -571,7 +571,7 @@ public boolean commitOffsets() {
         finishSuccessfulFlush();
         long durationMillis = time.milliseconds() - started;
         recordCommitSuccess(durationMillis);
-        log.info("{} Finished commitOffsets successfully in {} ms",
+        log.debug("{} Finished commitOffsets successfully in {} ms",
```

Comment:
nit: since this is mostly indicating exit from the method, I think it can be trace level too.

Reply:
I think you missed this one.